lib/resty/radixtree.lua

```diff
@@ -262,6 +401,212 @@ local function parse_remote_addr(route_remote_addrs)
     return ip_ins
 end
 
+local function pre_update_route(self, path, route, opts)
```

Comment:
These three functions: pre_insert_route, pre_modify_route, and pre_delete_route are 90% similar in the function body, please remove the duplicated code.

Reply:
new commit resolve it.